Bottom: 5a1dc0ad4e48638a32ff60dcbe1195c5a5e107a5
Top:    1020c581fcfc0ae347a261375e070e35513775f2
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-05 15:29:00 +0100

Refresh of hid-gtk-use-gtk_widget_get_sty

---

diff --git a/src/hid/gtk/gui-drc-window.c b/src/hid/gtk/gui-drc-window.c
index ce9a564..4ab38b1 100644
--- a/src/hid/gtk/gui-drc-window.c
+++ b/src/hid/gtk/gui-drc-window.c
@@ -752,7 +752,7 @@ ghid_violation_renderer_render (GtkCellRenderer      *cell,
 
   mydrawable = GDK_DRAWABLE (violation->pixmap);
 
-  gdk_draw_drawable (window, style->fg_gc[GTK_WIDGET_STATE (widget)],
+  gdk_draw_drawable (window, style->fg_gc[gtk_widget_get_state (widget)],
 		     mydrawable, 0, 0,
 		     cell_area->x + cell_area->width + VIOLATION_PIXMAP_PIXEL_BORDER,
 		     cell_area->y + VIOLATION_PIXMAP_PIXEL_BORDER, -1, -1);
